### PR TITLE
Added Obeliscus oris and supporting data

### PR DESCRIPTION
**Pull request by its-isha-singh**

### CODE REVIEW

File: virus/img/obeliscus_oris/.DS_Store
Comment (ababaian) [inline virus/img/obeliscus_oris/.DS_Store:1]: Please don't include dot files in the PR.

File: virus/Obeliscus_oris.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
+# _Obeliscus oris_: An Uncharacterised Circular RNA Encoding a Single Protein, Oblin-1
+written by: [Isha Singh](https://github.com/its-isha-singh)
+
+### _Obeliscus oris_
+_Obeliscus oris_ translates to _obelisk of the mouth_. All samples containing the
+obelisk (excluding one) were isolated from human oral-nasopharyngeal passages,
+hence the name _Obeliscus oris_.
+
+![](img/obeliscus_oris/cover.jpeg)
```
Comment (ababaian) [inline virus/Obeliscus_oris.md]: Please add "tittles" to the figures in the square brackets so text is displayed if hte image does not load.

**Review by ababaian — CHANGES_REQUESTED** [review]
:) Awesome! Can't wait to dive into this Obelisk with you. A couple of minor tweaks to close the PR.

File: virus/Obeliscus_oris.md (hunk 1 above)
Comment (its-isha-singh) [reply virus/Obeliscus_oris.md]: I've made the changes. Let me know if they went through!

**Review by ababaian — APPROVED** [review]

**Pull request merged**